### PR TITLE
SPEC-494 Specify behavior for command helpers accepting write concern in parameters

### PR DESCRIPTION
**Pull request by estolfo**

### CODE REVIEW

File: source/read-write-concern/read-write-concern.rst [hunk 1]
```diff
@@ -301,11 +301,13 @@ Other commands that write
 
 Command helper methods for commands that write, other than those discussed above,
 MAY accept a write concern or write concern options in their parameter list.
+If the helper accepts a write concern, the driver MUST error if the selected server's MaxWireVersion < 5 and a
+write concern has explicitly been specified.
```
Comment (jmikola) [inline source/read-write-concern/read-write-concern.rst]: Should this be specified for `findAndModify` as well? For any driver that hasn't already added an explicit "writeConcern" option but plan to introduce the functionality, they would do well to throw an error if the user specifies the option for MaxWireVersion < 4.

Comment (estolfo) [inline source/read-write-concern/read-write-concern.rst]: What do you think @craiggwilson ?
We can add some text saying a driver SHOULD raise an error if a user specifies a writeConcern to a findAndModify helper and the MaxWireVersion < 4.

Comment (behackett) [inline source/read-write-concern/read-write-concern.rst]: Does the server return an error if writeConcern is sent to older servers? If so, the driver doesn't necessarily have to do any validation. If the server silently ignores the field then the driver MUST raise an error.

Comment (estolfo) [inline source/read-write-concern/read-write-concern.rst]: Versions in which findAndModify does not support writeConcern (i.e. < 3.2), it is silently ignored.

Comment (estolfo) [inline source/read-write-concern/read-write-concern.rst]: I'm coming back to this now, I think we're ok to merge this specific change for 'Other commands that write', right?

As for findAndModify, Craig said he thinks it's fine to indicate to throw an error for < 3.2 so I'll add that text in.

**Comment by ajdavis** [comment]
This LGTM now.

**Comment by estolfo** [comment]
ok I'll merge it

**Pull request merged**